spec/fixtures/ai-proxy/openai/llm-v1-chat/requests/good_own_model.json

```diff
@@ -1,5 +1,5 @@
 {
-	"messages":[
+	"messages": [
```

Comment:
was hoping somebody would finally fix this

Reply:
I ran a giant linter over EVERYTHING xD